command_line/generate_tutorial_text.py

```diff
@@ -317,54 +311,54 @@ def extract_resolution(source, method):
 
 @dials.util.show_mail_handle_errors()
 def run(args=None):
-    parser = OptionParser(
+    parser = argparse.ArgumentParser(
         description="Generate tutorial logs for DIALS documentation website"
     )
-    parser.add_option("-?", action="help", help=SUPPRESS_HELP)
-    parser.add_option(
+    parser.add_argument("-?", action="help", help=argparse.SUPPRESS)
+    parser.add_argument(
         "--beta",
         dest="beta",
         action="store_true",
         default=False,
```

Comment:
```suggestion
        action="store_true",
```
These now not necessary in `ArgumentParser`